content/nextcloud.md

```diff
@@ -19,11 +19,44 @@ cloud experience (in the likes of Google Services, anyways).
 
 ## Instructions
 
-We should upgrade the system and then install packages that we might
-need. Run the following command:
+We should upgrade the system and then install the MariaDB server. Run the following command:
 
 ```sh
-apt full-upgrade -y && apt install mariadb-server php-mysql php php-gd php-mbstring php-dom php-curl php-zip php-simplexml php-xml php-fpm -y
+apt-get full-upgrade -y && apt-get install mariadb-server -y
+```
+
+In order for Nextcloud to operate properly, we need PHP version 7.4 or higher. Since Debian 10 does not include PHP 7.4, we need to add the community ran SURY repository which includes several versions of PHP for use on Debian 10.
+
+```sh
+apt -y install lsb-release apt-transport-https ca-certificates && wget -O /etc/apt/trusted.gpg.d/php.gpg https://packages.sury.org/php/apt.gpg && echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" | sudo tee /etc/apt/sources.list.d/php.list && apt update
```

Comment:
Nitpick: Mix of apt here and apt-get in other locations.  I'd use one or the other.

Reply:
Adapted to `apt`. Thanks!